E   AttributeError: module 'dns.rdtypes' has no attribute 'ANY'

Our weekly dependency check catched a new problem

```
tests/_test_mscolab/test_server.py:37: in <module>
    from mslib.mscolab.server import initialize_managers, check_login, register_user, APP
mslib/mscolab/server.py:151: in <module>
    _app, sockio, cm, fm = initialize_managers(APP)
mslib/mscolab/server.py:146: in initialize_managers
    sockio.init_app(app)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/flask_socketio/__init__.py:245: in init_app
    self.server = socketio.Server(**self.server_options)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/socketio/server.py:134: in __init__
    self.eio = self._engineio_server_class()(**engineio_options)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/engineio/server.py:139: in __init__
    self._async = importlib.import_module(
/opt/conda/envs/mssenv/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/engineio/async_drivers/eventlet.py:3: in <module>
    from eventlet.green.threading import Thread, Event
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/green/threading.py:2: in <module>
    import eventlet
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/__init__.py:17: in <module>
    from eventlet import convenience
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/convenience.py:7: in <module>
    from eventlet.green import socket
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/green/socket.py:21: in <module>
    from eventlet.support import greendns
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/support/greendns.py:66: in <module>
    setattr(dns, pkg, import_patched('dns.' + pkg))
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/support/greendns.py:61: in import_patched
    return patcher.import_patched(module_name, **modules)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/patcher.py:132: in import_patched
    return inject(
/opt/conda/envs/mssenv/lib/python3.10/site-packages/eventlet/patcher.py:109: in inject
    module = __import__(module_name, {}, {}, module_name.split('.')[:-1])
/opt/conda/envs/mssenv/lib/python3.10/site-packages/dns/zone.py:86: in <module>
    class Zone(dns.transaction.TransactionManager):
/opt/conda/envs/mssenv/lib/python3.10/site-packages/dns/zone.py:757: in Zone
    ) -> dns.rdtypes.ANY.SOA.SOA:
E   AttributeError: module 'dns.rdtypes' has no attribute 'ANY'
```

--- a/docs/mscolab.rst
+++ b/docs/mscolab.rst
@@ -188,7 +188,7 @@ Instructions to use mscolab wsgi
 make a file called :code:`server.py`
 and install ::
 
-   mamba install eventlet>0.30.2 gunicorn
+   mamba install eventlet\>0.30.2 dnspython\<2.3.0 gunicorn
 
 **server.py**::
 
@@ -196,7 +196,7 @@ and install ::
 
 Then run the following commands. ::
 
-  $ mamba install gunicorn eventlet>0.30.2
+  $ mamba install gunicorn eventlet\>0.30.2 dnspython\<2.3.0
   $ gunicorn -b 0.0.0.0:8087 server:app
 
 For further options read `<https://flask.palletsoperations.com/en/1.1.x/deploying/wsgi-standalone/#gunicorn>`_

--- a/requirements.d/development.txt
+++ b/requirements.d/development.txt
@@ -21,5 +21,5 @@ conda-verify
 Flask-Testing
 pytest-reverse
 eventlet>0.30.2
-dnspython>=2.0.0
+dnspython>=2.0.0, <2.3.0
 gsl==2.7.0

